### ISSUE TITLE
Fix all Svelte a11y warnings

### ISSUE THREAD

**Issue by eyeseast** [MuckRock/documentcloud-frontend#238]
Lots of clickable things that should be buttons. Here's every file emitting an a11y warning right now:

```sh
npm run build | grep 'WARNING in ./src' | uniq
WARNING in ./src/common/AccessToggle.svelte
WARNING in ./src/common/Autocomplete.svelte
WARNING in ./src/common/Calendar.svelte
WARNING in ./src/common/Dropdown.svelte
WARNING in ./src/common/FilePicker.svelte
WARNING in ./src/common/Image.svelte
WARNING in ./src/common/Modal.svelte
WARNING in ./src/common/ShareOptions.svelte
WARNING in ./src/common/dialog/AccessDialog.svelte
WARNING in ./src/common/dialog/CollaboratorDialog.svelte
WARNING in ./src/common/dialog/DataDialog.svelte
WARNING in ./src/common/dialog/EditSectionsDialog.svelte
WARNING in ./src/common/dialog/EmbedDialog.svelte
WARNING in ./src/pages/app/Anonymous.svelte
WARNING in ./src/pages/app/Document.svelte
WARNING in ./src/pages/app/File.svelte
WARNING in ./src/pages/app/SearchInput.svelte
WARNING in ./src/pages/embed/page/Note.svelte
WARNING in ./src/pages/embed/page/Page.svelte
WARNING in ./src/pages/entities/Entities.svelte
WARNING in ./src/pages/viewer/InternalPage.svelte
WARNING in ./src/pages/viewer/PageNoteInsert.svelte
WARNING in ./src/pages/viewer/SearchResults.svelte
WARNING in ./src/pages/viewer/Sidebar.svelte
WARNING in ./src/pages/viewer/ThumbnailBody.svelte
```

**Comment by duckduckgrayduck** [comment]
Not sure if this would be fixed with this, but I took a shot at using orca screen reader for Linux today on the DocumentCloud site on Chromium. When I hovered over drop-downs, selection boxes, etc I mostly would not get alerts to as to where I was on the page if I didn't click and drag to highlight it or if I didn't click on the item. Hover basically didn't do anything

**Comment by eyeseast** [comment]
It might be worth breaking some major chunks of the codebase out into dedicated issues. The menus need a lot of help. This might depend on whether we're redesigning them at all, too.
